orval.config.js

```diff
@@ -0,0 +1,17 @@
+import { defineConfig } from 'orval';
+
+export default defineConfig({
+    petstore: {
+        output: {
+            mode: 'tags-split',
+            target: 'src/badgehub-api-client/generated/swagger',
+            schemas: 'src/badgehub-api-client/generated/models',
+            client: 'fetch',
+            baseUrl: 'http://localhost:8001',
+        },
+
+        input: {
+            target: 'src/badgehub-api-client/swagger.json',
```

Comment:
Does this target field support a URL? I would really like it if the backend served the swagger file it's running and the codegen was based on that. We could even have the backend keep versioned swagger files available over http and then bump the version we point to here, rather than copying the file. It's not crucial, definitely not blocking.